RELEASE.md

```diff
@@ -1,36 +1,36 @@
-# Release Process
+### **OpenEBS Release Process Overview**
```

Comment:
Do we need bold on these titles?
Also why are we starting with `###` rather than `#` ?

Reply:
Right. Fixed it. I kept it the way it was for the bold titles.

Reply:
But why do we want bold titles on this particular document? We don't use it on the others?

Reply:
Yeah can remove that.